### DISCUSSION TITLE
Loading States should allow wire:loading to any display property

### DISCUSSION THREAD

**Discussion by gabbykweka** [livewire/livewire#4809] [3. Feature ideas]
I had a case showing spinner inside a table row for table loading period, but i could not make it display as I wished, because it suppose to use display:table-row. Which is not among the modifier available for wire:loading.

I think if this could have a dynamic way of setting CSS display property it would have been better for styling anything.

**Comment by silnarm** [comment]
Couldn't you just put that style on the element? Like, permanently. It will be hidden when not in the loading state anyway right, why do you need to toggle it? Are you using some other modifier on the `wire:loading` that change this behaviour? (`class` or `attr` I think are the only ones that do this).
That would also appear to be the solution if you are, toggle a `class` with your desired style.

**Reply by gabbykweka** [reply]
On showing event wire:loading is updating the display of element to its preference, not the one you set. And its not toggle, I have use table to display data, but on loading new data it suppose to use one of its row to display a spinner showing data is fetched.

**Reply by silnarm** [reply]
Struggling to make sense of that sorry.
Can you provide some code that demonstrates (removing everything not necessary to demonstrate)?
